src/com/magento/idea/magento2plugin/actions/CopyMagentoPath.java

```diff
@@ -22,10 +22,8 @@
 
 public class CopyMagentoPath extends CopyPathProvider {
     public static final String PHTML_EXTENSION = "phtml";
-    public static final String JS_EXTENSION = "js";
-    public static final String CSS_EXTENSION = "css";
-    private final List<String> acceptedTypes
-            = Arrays.asList(PHTML_EXTENSION, JS_EXTENSION, CSS_EXTENSION);
+    public static final List<String> WEB_EXTENSIONS
+            = Arrays.asList("js", "css", "png", "jpg", "jpeg", "gif");
```

Comment:
Just wondering, isn't possible to use [ImageIO::read](https://docs.oracle.com/javase/7/docs/api/javax/imageio/ImageIO.html#read%28java.io.File%29)? This should return `null` if the provided file cannot be read as an image. Or maybe there are some alternatives to check the file's `contentType`.

Reply:
I'm currently thinking of a way to support ONLY the Magento supported types. There's a lot more image types I missed which @DmitryFurs has described. So I'll check if the ImageIO types match the supported types, and develop a better solution. Thanks for the suggestion @eduard13 and @DmitryFurs!